src/Config/ConfigPanel.ts

```diff
@@ -88,10 +94,33 @@ export class ConfigPanel {
   }
 
   private _getHtmlForWebview(webview: vscode.Webview, context: vscode.ExtensionContext) {
-    // TODO Use a nonce to only allow specific scripts to be run
-    // TDOO fill with logics related to html
-    // TODO import html and change html source to fit webview format
-
-    return '';
+    const scriptUri =
+        webview.asWebviewUri(vscode.Uri.joinPath(this._extensionUri, 'media/Config', 'index.js'));
+
+    // Uri to load styles into webview
+    const stylesResetUri =
+        webview.asWebviewUri(vscode.Uri.joinPath(this._extensionUri, 'media/Config', 'reset.css'));
+    const stylesMainUri =
+        webview.asWebviewUri(vscode.Uri.joinPath(this._extensionUri, 'media/Config', 'vscode.css'));
+
+    // Use a nonce to only allow specific scripts to be run
+    const nonce = getNonce();
+
+    // Get html file for webview
+    const filePath: vscode.Uri =
+        vscode.Uri.file(path.join(context.extensionPath, 'media', 'Config', 'index.html'));
+    let html = fs.readFileSync(filePath.fsPath, 'utf8');
+    let re = /\${webview.cspSource}/gi;
+    html = html.replace(re, `${webview.cspSource}`);
+    re = /\${stylesResetUri}/gi;
+    html = html.replace(re, `${stylesResetUri}`);
+    re = /\${stylesMainUri}/gi;
+    html = html.replace(re, `${stylesMainUri}`);
+    re = /\${scriptUri}/gi;
+    html = html.replace(re, `${scriptUri}`);
+    re = /\${nonce}/gi;
+    html = html.replace(re, `${nonce}`);
+    html = html.replace(re, `${nonce}`);
```

Comment:
is there some problem calling same code twice ?

Reply:
Because replace function replace word only for once. but in media/Config/index.html has two ${nonce} part.
That is the reason

Reply:
Then, can you add a comment above these two like
```
// NOTE there exit two replacements in 'index.html' for 'nonce'
```

Reply:
@OTKRyu Did you try above codes? Since `re` is regex pattern, just calling `replace` once will do the trick:)

Reply:
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/String/replace shows only first is replaced

Reply:
`g` applies to all the words. Please try with `g`.

Reply:
it works well. thank you for your advice